.github/workflows/generators.yml

```
@@ -25,7 +25,7 @@ jobs:
       - name: Install non-Python dependencies
         run: |
           apt update
-          apt install -y pkg-config autoconf libtool gcc make
+          apt install -y pkg-config autoconf libtool gcc g++ make
```

Why not just [`build-essential`](https://packages.ubuntu.com/jammy/build-essential)?

---

```suggestion
          apt install -y pkg-config autoconf libtool build-essential
```
Usually you need them altogether (that's why they made a metapackage).